http/src/main/resources/init-resources/rstudio-docker-compose.yaml

```diff
@@ -22,6 +22,7 @@ services:
       - /etc/rstudio-license-file.lic:/etc/rstudio-server/license.lic:ro
     env_file:
       - /etc/google_application_credentials.env
+      - /etc/custom_env_vars.env
```

Comment:
Needed to support custom EVs in the RStudio image. This already exists in the Jupyter image, was just missed here I think.